src/constants/index.tsx

```diff
@@ -252,6 +252,8 @@ export const ALLOWED_PRICE_IMPACT_HIGH: Percent = new Percent(JSBI.BigInt(500),
 export const PRICE_IMPACT_WITHOUT_FEE_CONFIRM_MIN: Percent = new Percent(JSBI.BigInt(1000), BIPS_BASE) // 10%
 // for non expert mode disable swaps above this
 export const BLOCKED_PRICE_IMPACT_NON_EXPERT: Percent = new Percent(JSBI.BigInt(1500), BIPS_BASE) // 15%
+// used for fiat warning states
+export const ALLOWED_FIAT_PRICE_IMPACT_HIGH: Percent = new Percent(JSBI.BigInt(200), BIPS_BASE) // 2%
```

Comment:
I would say 2% is high. We will get back to this when @zamli is back.

Reply:
I don't know if this helps but in the issue is defined 2% for "high price impact". https://github.com/levelkdev/dxswap-dapp/issues/640